<?xml version="1.0" encoding="utf-8"?><messages><msg code="0" type="2">Start Time: Thursday, 10 July 2025 16:28:00</msg><msg code="-2147467259" type="100">﻿Traceback (most recent call last):
﻿  File "E:\git\LevellingThesis\Code\FindMaximalLoop.py", line 36, in &lt;module&gt;
﻿    find_max_loop(lines_input, points_input, max_loop_output)
﻿  File "E:\git\LevellingThesis\Code\FindMaximalLoop.py", line 20, in find_max_loop
﻿    find_loops(selected_lines, points_input, 'independent', max_loop_output)
﻿  File "E:\git\LevellingThesis\Code\FindLoops.py", line 41, in find_loops
﻿    polygons_data = extract_data_from_cycles(G)
﻿                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^
﻿  File "E:\git\LevellingThesis\Code\Utils\Loops.py", line 157, in extract_data_from_cycles
﻿    g_corr += grav_corr_value
﻿TypeError: unsupported operand type(s) for +=: 'int' and 'NoneType'
</msg><msg code="0" type="0">Failed script Find Maximal Loop...</msg><msg code="-2147467259" type="100">Failed to execute (FindMaximalLoop).</msg><msg code="0" type="3">Failed at Thursday, 10 July 2025 16:28:05 (Elapsed Time: 5.17 seconds)</msg></messages>